src/components/Table/Table/__stories__/table.stories.mdx

```diff
@@ -0,0 +1,187 @@
+import { useState } from "react";
+import Table from "../Table";
+import TableHeader from "../../TableHeader/TableHeader";
+import TableHeaderCell from "../../TableHeaderCell/TableHeaderCell";
+import TableBody from "../../TableBody/TableBody";
+import TableVirtualizedBody from "../../TableVirtualizedBody/TableVirtualizedBody";
+import TableRow from "../../TableRow/TableRow";
+import TableCell from "../../TableCell/TableCell";
+import Label from "../../../Label/Label";
+import Skeleton from "../../../Skeleton/Skeleton";
+import Flex from "../../../Flex/Flex";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { createStoryMetaSettingsDecorator } from "../../../../storybook";
+import { Link, MultipleStoryElementsWrapper, createComponentTemplate } from "vibe-storybook-components";
+import { emailTableData, emailColumns, TableAvatar, TableErrorState, TableEmptyState, sort, basicTableProps, scrollTableColumns, scrollTableData } from './helpers';
+
+export const metaSettings = createStoryMetaSettingsDecorator({
+  component: Table,
+})
+
+<Meta
+  title="Data display/Table"
+  component={ Table }
+  argTypes={ metaSettings.argTypes }
+  decorators={ metaSettings.decorators }
+/>
+
+<!--- Component template -->
+
+export const tableTemplate = createComponentTemplate(Table);
+
+<!--- Component documentation -->
+
+# Table
+- [Overview](#overview)
+- [Props](#props)
+- [Loading](#loading)
+- [Empty State](#empty-state)
+- [Scroll](#scroll)
+
+## Overview
+Table
+<Canvas>
+  <Story name="Overview">
+    {() => {
+      const [tableData, setTableData] = useState(emailTableData);
+      const [sorting, setSorting] = useState({});
+      const onSort = (columnId, sortState) => {
+        setSorting({ [columnId]: sortState });
+        setTableData(sort(columnId, sortState, tableData));
+      };
+      return (
+        <Table
+          dataState={{ isError: false, hasMoreResults: false }}
```

Comment:
I don't think this prop should be required. We can implement smart defaults to reduce the amount of boilerplate the developer needs to write